### PR TITLE
[nl-NL] Translate Police Exam to Dutch.

### PR DESCRIPTION
**Pull request by xBytez**
Additional:
[en-US] Change formatting to end with ~s~ for police exam questions.

### CODE REVIEW

File: jobs.xml [hunk 1]
```diff
@@ -2742,255 +2742,315 @@
...
     </Entry>
 
     <Entry Id="police_exam_answer_lethal_force_respect">
         <String xml:lang="en-US">To teach the ~o~suspect ~s~some respect</String>
+        <String xml:lang="nl-NL">Om de ~o~verdachte ~s~een beetje respect te leren</String>
     </Entry>
 
     <Entry Id="police_exam_answer_lethal_force_most_wanted">
         <String xml:lang="en-US">To take down a ~r~most wanted ~s~felon</String>
+        <String xml:lang="nl-NL">Om een ~r~meest gezochte ~s~verdacht neer te halen</String>
```
Comment (stijnjw) [inline jobs.xml:2785]: verdachte

Comment (stijnjw) [reply jobs.xml:2785]: En effe kijken naar consistency teggen meest gezocht en bounty

File: jobs.xml [hunk 2]
```diff
@@ -2742,255 +2742,315 @@
...
     </Entry>
 
     <Entry Id="police_exam_answer_tires_mw">
         <String xml:lang="en-US">When the suspect is ~r~most wanted~s~</String>
+        <String xml:lang="nl-NL">Wanneer de verdachte een ~r~bounty~s~ heeft</String>
     </Entry>
 
     <Entry Id="police_exam_answer_tires_ramming">
         <String xml:lang="en-US">When the suspect is ramming their patrol car that they are currently occupying</String>
+        <String xml:lang="nl-NL">Wanneer de verdachte het dienstvoertig ramt waar de agent in zit</String>
```
Comment (stijnjw) [inline jobs.xml:2815]: dienstvoertuig

**Review by stijnjw — CHANGES_REQUESTED** [review]

File: jobs.xml (hunk 2 above)
Comment (xBytez) [reply jobs.xml]: Scherp! Fixed in 661541e

**Review by stijnjw — APPROVED** [review]

**Pull request merged**